### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by sammou00** [HYF-Class19/agile-development-loruki-group-4#24] [General]
Review

**Comment by sammou00** [comment]
Hello, well done.
Some notes :
1- project deadline is an external constrains
![image](https://user-images.githubusercontent.com/67711948/182948267-ebb98dae-c83d-4d80-ae5c-10fa85e4cf8a.png)
 
2- good `user-personas` 
3- I think the `title` is missing in your `backlog` 
![image](https://user-images.githubusercontent.com/67711948/182948627-13544caa-7211-4c3f-a604-23b68a22d879.png)

4 - the title existed in your `strategy-plan` , but not in your `backlog` 
![image](https://user-images.githubusercontent.com/67711948/182948763-fc179f57-261d-4ba7-97cd-fadfc022980b.png)

5- very good `index.html` with nice and well organized comments 
![image](https://user-images.githubusercontent.com/67711948/182949038-20a6cb9b-0707-4478-8498-19ca914d48c7.png)

6- also very good and organized `index.css`, maybe add general styles as a comment in the top of your general styles 
![image](https://user-images.githubusercontent.com/67711948/182949534-71190435-d6bc-43b5-a2c5-e2d276b003b7.png)

**Reply by AzimaKai** [reply]
Thank You very much for attention to our group 🤗 We will take this notes and improve it better 🙏 🙏 🙏
